shop/src/main/resources/db/migration/V10__first_insert_migration.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO PRODUCTCATEGORY(NAME, DESCRIPTION) VALUES ( 'laptop', 'this is laptop category' );
```

Comment:
You should create a specific migration file for the main operations here (eg. V1 - table creation, V2-populate tables etc). You have too many major version migration files. Imagine having double the number of tables you have now.